app/jobs/send_request_events_to_bigquery.rb

```diff
@@ -0,0 +1,10 @@
+class SendRequestEventsToBigquery < ActiveJob::Base
+  TABLE_NAME = 'events'.freeze
```

Comment:
We're going to change Find and probably Apply to make this a configuration point. I'd suggest using `Settings.google.big_query_table`